### DISCUSSION TITLE
What is the definition of Unattended?

### DISCUSSION THREAD

**Discussion by harismateen** [chatwoot/chatwoot#8308] [Q&A]
Hi,
What is the definition of Unattended?
![image](https://github.com/chatwoot/chatwoot/assets/38554929/c0658dec-9316-4f72-937f-3833e9dc956f)

**Comment by pranavrajs** [comment]
Unattended conversations are the ones that are missing a reply from the customer support team. This number shows the conversations that are missing a first reply or a followup reply.

**Reply by harismateen** [reply]
@pranavrajs thanks - is there away to see list of the conversations are unattended? somehow, i tried to recreate the open conversations but the count didn't reflect.
